packages/@lightningjs/ui-components/src/components/TextBox/TextBox.stories.js

```diff
@@ -71,6 +75,22 @@ Basic.argTypes = {
     table: {
       defaultValue: { summary: false }
     }
+  },
+  fixed: {
+    control: 'boolean',
+    description:
+      'Flag that when set to `true`, allows the width of the component to be set with `w`',
+    table: {
+      defaultValue: { summary: 'undefined' }
```

Comment:
Minor one, I think we should change defaultValue for fixed control to boolean.

Reply:
If we don't set `fixed`, it technically is 'undefined' (console logging shows 'undefined') so that's why I put 'undefined' as the default. However, since it is supposed to be a boolean, is it better to show the default as 'false'?

Reply:
You are right , but to be consistent I would expect to show the default value as false here.
I think in the Button component for fixed control the default value is false.